src/api/namespaces/neftydrops/routes/marketplace.ts

```diff
@@ -141,6 +143,33 @@ export function marketplaceEndpoints(core: NeftyDropsNamespace, server: HTTPServ
             items: {'$ref': '#/components/schemas/CollectionsBalance'}
           })
         }
+      },
+      '/v1/marketplace/trading/volume-and-average': {
+        get: {
+          tags: ['marketplace'],
+          summary: 'Get total trading volume and reward average',
+          description: 'Get total trading volume and reward average.\n' +
+            "Total trading volume is 'amount of wax spent in nefty market' + " + 
+            "'amount of wax sold in nefty market' + 'amount of wax and nefty spent buying drops'\n" + 
+            "Reward average is the average NEFTY reward per wallet (0 if no wallets " +
+            "were rewarded in that period of time)",
+          parameters: [
+            {
+              name: 'total_nefty_reward',
+              in: 'query',
+              description: 'How much nefty will be rewarded in the given time range, used to calculate the Reward average',
+              required: false,
+              schema: {
+                type: 'float'
+              }
+            },
+            ...dateBoundaryParameters,
+          ],
+          responses: getOpenAPI3Responses([200, 500], {
+            type: 'array',
+            items: {'$ref': '#/components/schemas/CollectionsBalance'}
```

Comment:
this is not a `CollectionsBalance` so if you can create later an appropriate schema that would be nice